.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Let's not commit this please unless you have a very good reason
https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file